### PR TITLE
chore: share core styles and configs [DET-7316]

### PR DESCRIPTION
**Pull request by hamidzr**
## Description

- share all or parts of tsconfig, eslintrc, stylelintrc, craco
- share ant style overrides and our themes

## Test Plan

- bulid and visually check and expect no differences

## Commentary (optional)

assuming this is good with the reviewers I'll move on to pulling this into saas.

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: webui/react/src/components/Logo.tsx [hunk 1]
```diff
@@ -12,9 +12,9 @@ import logoDeterminedOnLightVertical from
   'shared/assets/images/logo-determined-on-light-vertical.svg';
 import logoHpeOnDarkHorizontal from 'shared/assets/images/logo-hpe-on-dark-horizontal.svg';
 import logoHpeOnLightHorizontal from 'shared/assets/images/logo-hpe-on-light-horizontal.svg';
-import { DarkLight } from 'themes';
 import { BrandingType } from 'types';
 
+import { DarkLight } from '../shared/themes';
```
Comment (hkang1) [inline webui/react/src/components/Logo.tsx:17]: **suggestion:** `shared/themes` ("absolute" instead of relative path)

**Review by hkang1 — APPROVED** [review]
LGTM!

File: webui/react/src/components/Logo.tsx (hunk 1 above)
Comment (hamidzr) [reply webui/react/src/components/Logo.tsx:17]: these are how webstorm puts them (automated) I'm thinking of writing a script or finding a tool that updates these for us.

Comment (hamidzr) [reply webui/react/src/components/Logo.tsx:17]: (already included as part of [this ticket ](https://determinedai.atlassian.net/browse/DET-7628?atlOrigin=eyJpIjoiMjliZDhjZmFmZGNjNDQ4OGExNTM3NDYzNjkwMGVmZjYiLCJwIjoiamlyYS1zbGFjay1pbnQifQ) we have other instances of this

File: webui/react/src/shared/bin/test.py [hunk 2]
```diff
@@ -6,7 +6,6 @@
```
Comment (hamidzr) [inline webui/react/src/shared/bin/test.py:6]: this and the readme change jsut revert changes that probably should have been added to shared-web first?

**Pull request merged**